Better Transformer integration (#423)
huggingface/optimum: docs/source/bettertransformer/quickstart.mdx

specific language governing permissions and limitations under the License.
-->

# Quickstart

Since its 1.13 version, PyTorch released the stable version of `BetterTransformer` in its library. You can benefit from interesting speedup on most consumer-type devices, including CPUs, older and newer versions of NIVIDIA GPUs.
You can now use this feature in HuggingFace `optimum` together with `transformers` and use it for major models in the HuggingFace ecosystem.

## Supported models

The list of supported model below:
- [AlBERT](https://arxiv.org/abs/1909.11942)
- [BERT](https://arxiv.org/abs/1810.04805)
- [Electra](https://arxiv.org/abs/2003.10555)
- [MarkupLM](https://arxiv.org/abs/2110.08518)
- [Data2VecText](https://arxiv.org/abs/2202.03555)
- [CamemBERT](https://arxiv.org/abs/1911.03894)
- [RoBERTa](https://arxiv.org/abs/1907.11692)
- [Splinter](https://arxiv.org/abs/2101.00438)
- [Ernie](https://arxiv.org/abs/1904.09223)
- [LayoutLM](https://arxiv.org/abs/1912.13318)
- [BART](https://arxiv.org/abs/1910.13461)
- [DistillBert](https://arxiv.org/abs/1910.01108)
- [XLMRoberta](https://arxiv.org/abs/1911.02116)

> Let us know by opening an issue in `optimum` if you want more models to be supported!

Author: ```suggestion
Let us know by opening an issue in `optimum` if you want more models to be supported, or check out the contribution guideline if you want to add it by yoursefl!
```

> <!--Copyright 2022 The HuggingFace Team. All rights reserved.

Reviewer: Could we move this in overview.mdx, maybe?

Author: Thanks ! Fixed in 9d5f90a

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
specific language governing permissions and limitations under the License.
-->

# Quickstart

Since its 1.13 version, PyTorch released the stable version of `BetterTransformer` in its library. You can benefit from interesting speedup on most consumer-type devices, including CPUs, older and newer versions of NIVIDIA GPUs.

> You can now use this feature in HuggingFace `optimum` together with `transformers` and use it for major models in the HuggingFace ecosystem.

Reviewer: ```suggestion
You can now use this feature in 🤗 Optimum together with Transformers and use it for major models in the Hugging Face ecosystem.
```

Reviewer: just to keep it consistent with Transformers doc

## Supported models

The list of supported model below:

> - [AlBERT](https://arxiv.org/abs/1909.11942)
> - [BERT](https://arxiv.org/abs/1810.04805)
> - [Electra](https://arxiv.org/abs/2003.10555)
> - [MarkupLM](https://arxiv.org/abs/2110.08518)
> - [Data2VecText](https://arxiv.org/abs/2202.03555)
> - [CamemBERT](https://arxiv.org/abs/1911.03894)
> - [RoBERTa](https://arxiv.org/abs/1907.11692)
> - [Splinter](https://arxiv.org/abs/2101.00438)
> - [Ernie](https://arxiv.org/abs/1904.09223)
> - [LayoutLM](https://arxiv.org/abs/1912.13318)
> - [BART](https://arxiv.org/abs/1910.13461)
> - [DistillBert](https://arxiv.org/abs/1910.01108)
> - [XLMRoberta](https://arxiv.org/abs/1911.02116)
> - [Whisper](https://cdn.openai.com/papers/whisper.pdf)

Reviewer: ```suggestion
- [AlBERT](https://arxiv.org/abs/1909.11942)
- [BART](https://arxiv.org/abs/1910.13461)
- [BERT](https://arxiv.org/abs/1810.04805)
- [CamemBERT](https://arxiv.org/abs/1911.03894)
- [Data2VecText](https://arxiv.org/abs/2202.03555)
- [DistilBert](https://arxiv.org/abs/1910.01108)
- [Electra](https://arxiv.org/abs/2003.10555)
- [Ernie](https://arxiv.org/abs/1904.09223)
- [LayoutLM](https://arxiv.org/abs/1912.13318)
- [MarkupLM](https://arxiv.org/abs/2110.08518)
- [RoBERTa](https://arxiv.org/abs/1907.11692)
- [Splinter](https://arxiv.org/abs/2101.00438)
- [XLMRoberta](https://arxiv.org/abs/1911.02116)
- [Whisper](https://cdn.openai.com/papers/whisper.pdf)
```

Let us know by opening an issue in `optimum` if you want more models to be supported, or check out the contribution guideline if you want to add it by yoursefl!

## Quick usage

In order to use the `BetterTransformer` API just run the following commands:

```python
>>> from transformers import AutoModelForSequenceClassification
>>> from optimum.bettertransformer import BetterTransformer

> 
> 

Reviewer: ```suggestion

```
